cpp/include/cudf/fixed_point/fixed_point.hpp

```diff
@@ -283,8 +283,8 @@ struct scaled_integer {
  */
 template <typename Rep, Radix Rad>
 class fixed_point {
-  Rep _value;
-  scale_type _scale;
+  Rep _value{};
```

Comment:
Should this be `_value{0}` ?

Reply:
They are equivalent. Using `{}` is more flexible in case `Rep` is something more complex than just `int` types.